dist/index.html

```diff
@@ -0,0 +1 @@
+<!doctype html><html lang="en"><head><meta charset="UTF-8"/><meta http-equiv="X-UA-Compatible" content="IE=edge"/><meta name="viewport" content="width=device-width,initial-scale=1"/><title>Document</title><script defer="defer" src="main.bundle.js"></script><link href="main.50430588118f71f34582.css" rel="stylesheet"></head><body><div id="root"></div></body></html>
```

Comment:
@AStashevskaya Lets add `dist` directory to `.gitignore` and use `build` script which should be automatically runned on heroku deploy